lib/spoom/cli/bump.rb

```diff
@@ -146,7 +146,7 @@ def print_changes(files, command:, from: "false", to: "true", dry: false, path:
             say(" + #{file_path}")
           end
           if dry && command
-            say("\nRun `#{command}` to bump them")
+            say("\nRun `#{command}` to bump #{files.length > 1 ? "them" : "it"}")
```

Comment:
Thanks.

Do you mind changing [line 141](https://github.com/Shopify/spoom/blob/9a687f707a85336a86ce6bc930d3dd1421dff642/lib/spoom/cli/bump.rb#L141) to also use `length` so we keep some uniformity please?

Reply:
Done! Thanks for the review!